2024-01-31 20:44:23,627 [INFO] - C:\Users\mistr\captcha-champion\captcha-backend\captcha_controller\game\views.py changed, reloading.
2024-01-31 20:44:36,379 [INFO] - C:\Users\mistr\captcha-champion\captcha-backend\captcha_controller\game\views.py changed, reloading.
2024-01-31 20:44:50,324 [ERROR] - !! Exception in game views for selecting the plan'CaptchaPlanRecord' object has no attribute 'is_active'
2024-01-31 20:44:50,326 [WARNING] - Bad Request: /api/select-plan/
2024-01-31 20:46:10,715 [INFO] - C:\Users\mistr\captcha-champion\captcha-backend\captcha_controller\game\views.py changed, reloading.
2024-01-31 20:46:14,553 [ERROR] - An exception occurred: 'CaptchaPlanRecord' object has no attribute 'is_active'
Traceback (most recent call last):
  File "C:\Users\mistr\captcha-champion\captcha-backend\captcha_controller\game\views.py", line 47, in select_plan
    CaptchaPlanRecord.objects.create(user=user,plan=selected_plan)
  File "C:\Users\mistr\captcha-champion\captcha-backend\myvenv\lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\mistr\captcha-champion\captcha-backend\myvenv\lib\site-packages\django\db\models\query.py", line 677, in create
    obj.save(force_insert=True, using=self.db)
  File "C:\Users\mistr\captcha-champion\captcha-backend\myvenv\lib\site-packages\django\db\models\base.py", line 822, in save
    self.save_base(
  File "C:\Users\mistr\captcha-champion\captcha-backend\myvenv\lib\site-packages\django\db\models\base.py", line 924, in save_base
    post_save.send(
  File "C:\Users\mistr\captcha-champion\captcha-backend\myvenv\lib\site-packages\django\dispatch\dispatcher.py", line 189, in send
    response = receiver(signal=self, sender=sender, **named)
  File "C:\Users\mistr\captcha-champion\captcha-backend\captcha_controller\game\models.py", line 42, in ensure_single_active_plan
    if instance.is_active:
AttributeError: 'CaptchaPlanRecord' object has no attribute 'is_active'
2024-01-31 20:46:14,607 [WARNING] - Bad Request: /api/select-plan/
2024-01-31 20:46:31,035 [INFO] - C:\Users\mistr\captcha-champion\captcha-backend\captcha_controller\game\views.py changed, reloading.
2024-01-31 20:46:41,726 [INFO] - C:\Users\mistr\captcha-champion\captcha-backend\captcha_controller\game\views.py changed, reloading.
2024-01-31 20:46:43,874 [INFO] - C:\Users\mistr\captcha-champion\captcha-backend\captcha_controller\game\views.py changed, reloading.
2024-01-31 20:46:52,277 [INFO] - C:\Users\mistr\captcha-champion\captcha-backend\captcha_controller\game\views.py changed, reloading.
2024-01-31 20:46:54,712 [ERROR] - An exception occurred: 'CaptchaPlanRecord' object has no attribute 'is_active'
Traceback (most recent call last):
  File "C:\Users\mistr\captcha-champion\captcha-backend\captcha_controller\game\views.py", line 47, in select_plan
    CaptchaPlanRecord.objects.create(user=user,plan=selected_plan)
  File "C:\Users\mistr\captcha-champion\captcha-backend\myvenv\lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\mistr\captcha-champion\captcha-backend\myvenv\lib\site-packages\django\db\models\query.py", line 677, in create
    obj.save(force_insert=True, using=self.db)
  File "C:\Users\mistr\captcha-champion\captcha-backend\myvenv\lib\site-packages\django\db\models\base.py", line 822, in save
    self.save_base(
  File "C:\Users\mistr\captcha-champion\captcha-backend\myvenv\lib\site-packages\django\db\models\base.py", line 924, in save_base
    post_save.send(
  File "C:\Users\mistr\captcha-champion\captcha-backend\myvenv\lib\site-packages\django\dispatch\dispatcher.py", line 189, in send
    response = receiver(signal=self, sender=sender, **named)
  File "C:\Users\mistr\captcha-champion\captcha-backend\captcha_controller\game\models.py", line 42, in ensure_single_active_plan
    if instance.is_active:
AttributeError: 'CaptchaPlanRecord' object has no attribute 'is_active'
2024-01-31 20:46:54,718 [WARNING] - Bad Request: /api/select-plan/
2024-01-31 20:47:55,053 [INFO] - C:\Users\mistr\captcha-champion\captcha-backend\captcha_controller\game\views.py changed, reloading.
2024-01-31 20:48:08,871 [INFO] - C:\Users\mistr\captcha-champion\captcha-backend\captcha_controller\game\views.py changed, reloading.
